pkg/wallet/wallet.go

```diff
@@ -224,8 +224,17 @@ func Delete(nickname string) (err error) {
 	return nil
 }
 
-func CheckAddress(address string) bool {
-	return len(address) > MinAddressLength
+func (w *Wallet) CheckAddressValidity(address string) bool {
```

Comment:
Since the wallet code is being moved to a plugin I'm not sure we should put this code here

Reply:
Yeah but we do not want to make a network call everytime we want to check an address format, so this piece of code won't live as a plugin

Reply:
In fact we didn't need this actually so i will delete it